components/base/kinematicbase/ptgKinematics_test.go

```diff
@@ -21,9 +21,7 @@ import (
 func TestPTGKinematicsNoGeom(t *testing.T) {
 	logger := logging.NewTestLogger(t)
 
-	name, err := resource.NewFromString("is:a:fakebase")
```

Comment:
You can no longer use `NewFromString` for a non-fully-qualified resource name (this one is missing a `/`). Manually created a `resource.Name` here and below instead.